Python/Setup/loclayout.proj

```diff
@@ -23,6 +23,7 @@
         Microsoft.PythonTools.Debugger.Concord.dll;
         Microsoft.PythonTools.dll;
         Microsoft.PythonTools.EnvironmentsList.dll;
+        Microsoft.PythonTools.LiveShare.dll;
```

Comment:
Actually I should remove these. Live share isn't needed anymore with pylance as VS will handle the negotiations for it.